knowledge_repo/config.py

```diff
@@ -1,5 +1,3 @@
-from .constants import PY_EXTENSION, YML_EXTENSION
-from .utils.files import read_yaml
```

Comment:
Let's keep all import statements sorted in alphabetical order for easy maintenance.